src/javascript/content-editor/view.js

```diff
@@ -288,10 +288,13 @@ ContentEditor.prototype.sendPostEvent = function (ev) {
             return ev.failure(err);
         }
         ev.success();
-        self._uploadButton || self._addUploadButton();
```

Comment:
This is fine because `_addUploadButton` doesn't create a new one if it already exists.